src/NBitcoin/IConsensus.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using NBitcoin.BouncyCastle.Math;
+
+namespace NBitcoin
+{
+    public interface IConsensus
+    {
+        /// <summary>
+        /// How many blocks should be on top of a coinbase transaction until its outputs are considered spendable.
```

Comment:
on top of a block that includes a coinbase transaction

Reply:
@rowandh can you fix the comment here?